Plugin.php

```diff
@@ -64,7 +65,8 @@ public function registerPermissions()
      */
     public function boot()
     {
-        $monolog = Log::getMonolog();
+        $laravel8 = method_exists(Writer::class, 'getLogger');
```

Comment:
This line is always false, because `\Illuminate\Log\Writer` is removed since Laravel 5.6, no matter what Laravel version is.

I suggest to use `\Illuminate\Log\Logger` instead.

Then, `Logger` class name conflicts with `Monolog\Logger` included at the top and used as return type in later methods. For this, I suggest to replace `Monolog\Logger` with `Psr\Log\LoggerInterface` which both `Illuminate\Log\Logger` and `Monolog\Logger` implement.

With these changes, it is working in both of my October projects with Laravel 5.5 and Laravel 6.